src/formatters/captureChoiceFormatter.ts

```diff
@@ -171,8 +171,12 @@ export class CaptureChoiceFormatter extends CompleteFormatter {
 			return -1;
 		}
 
-		if (fileCache.frontmatter.position)
-			return fileCache.frontmatter.position.end.line;
+		if (fileCache.frontmatter.position || fileCache.frontmatterPosition)
+			if (fileCache.frontmatter.position) {
+				return fileCache.frontmatter.position.end.line;
+			} else {
+				return fileCache.frontmatterPosition.end.line;
+			}
```

Comment:
Per suggestion in https://github.com/obsidianmd/obsidian-api/issues/122#issuecomment-1664346884. Also cleans up code a bit.
```suggestion
        const frontmatterPosition = fileCache.frontmatterPosition || fileCache.frontmatter?.position
        if (frontmatterPosition) {
            return frontmatterPosition.end.line;
```

Reply:
Thanks! Applied to the PR.